lib/columns_trace/created_record.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ColumnsTrace
+  # Class that is used to store metadata about created ActiveRecord records.
+  class CreatedRecord
+    attr_reader :model, :record, :backtrace
+
+    def initialize(record, backtrace)
+      @model = record.class
+      @record = record
+      @backtrace = backtrace
+    end
+
+    def accessed_fields
+      record.accessed_fields
+    end
+
+    def unused_fields
+      # We need to store this into local variable, because `record.attributes`
+      # will access all attributes.
```

Comment:
Do we need to call `accessed_fields` before `unused_fields` to use it right, or order does not matter?

Reply:
It should not matter, I push a small fix for this to actually be the case.